src/BuildASub.java

```diff
@@ -60,4 +68,43 @@ public static String chooseMeat() throws Exception {
             throw new Exception("Meat not set!");
         }
     }
+
+    public static String ChooseCheese(){
+        String SelectedCheese = "";
+        System.out.println("What cheese do you want: \n 1. Regular \n 2. Peppered");
+        Scanner scan = new Scanner(System.in);
+        int CheeseOption = scan.nextInt();
+        switch (CheeseOption){
+            case 1:
+                SelectedCheese = "Regular";
+                        break;
```

Comment:
the `break;` call should be at the same level of tabs as `SelectedCheese`. This should have been picked up in my first commit too!